src/index.html

```diff
@@ -11,40 +11,40 @@
   <body>
     <header>
       <h1><span>fresh</span>Picks<img src="../images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
-      <form id="search" role="search">
+      <nav id="search" role="search">
```

Comment:
We might want to double check this. I'm not sure if nav will perform the same way that form performs, and it might mess with some of our css functionality when it comes time to start working with it.